Not compatible with React v18 (Next.js v12)

When trying to install with `next@12.2.2` there's a peer dependencies error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: website@0.1.0
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"18.2.0" from the root project
npm ERR!   peer react@"^17.0.2 || ^18.0.0-0" from next@12.2.2
npm ERR!   node_modules/next
npm ERR!     next@"12.2.2" from the root project
npm ERR!     peer next@"^10 || ^11 || ^12" from @panelbear/panelbear-nextjs@1.0.3
npm ERR!     node_modules/@panelbear/panelbear-nextjs
npm ERR!       @panelbear/panelbear-nextjs@"*" from the root project
npm ERR!   1 more (react-dom)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16 || ^17" from @panelbear/panelbear-nextjs@1.0.3
npm ERR! node_modules/@panelbear/panelbear-nextjs
npm ERR!   @panelbear/panelbear-nextjs@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/codespace/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/codespace/.npm/_logs/2022-08-03T14_13_31_258Z-debug-0.log
```

The installation can be forced with `--legacy-peer-deps`

--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
   },
   "peerDependencies": {
     "next": "^10 || ^11 || ^12",
-    "react": "^16 || ^17"
+    "react": "^16 || ^17 || ^18"
   },
   "dependencies": {
     "@panelbear/panelbear-js": "^1.3.3"

